docs: v4.0.0 cleanup (#1260)
exadel-inc/esl: src/modules/esl-traversing-query/README.md

- `::parent::child(some-tag)` - find direct child element(s) that match tag 'some-tag' in the parent
- `#id .class [attr]::parent` - find parent of element matching selector '#id .class [attr]' in document
- `::find(.row)::last::parent` - find parent of the last element matching selector '.row' from the base element subtree

### API

Find one (first element that describes a query):
```typescript

>   ESLTraversingQuery.first(query: string, base?: Element, scope?: Element): Element | null;

Reviewer: ```suggestion
  ESLTraversingQuery.first(query: string, base?: Element, scope?: Element | Document): Element | null;
```

```

Find all elements that describes a query:
```typescript

>   ESLTraversingQuery.all(query: string, base?: Element, scope?: Element): Element[];

Reviewer: ```suggestion
  ESLTraversingQuery.all(query: string, base?: Element, scope?: Element | Document): Element[];
```

```

Params: 

> - `query` - query sting
> - `base` - describes a base for relative query.
> - `scope` - define a scope to find global query (`document` by default)

Reviewer 2: ```suggestion
- `query` - query string
- `base` - describes a base for relative query.
- `scope` - defines a scope to find global query (`document` by default)
```
